index.html

```diff
@@ -531,6 +531,16 @@ <h2>
       <h2>
 Services
       </h2>
+      <p>
+Service descriptions are used in DID Documents to describe ways in which one can
+communicate with the subject of the DID or associated entities. Services listed
+in the DID Document can contain information about privacy preserving messaging
+services, or more public information such as social media accounts, personal
+websites, and email addresses. The metadata  associated with services are often
+service specific. For example, an encrypted messaging service most likley also
```

Comment:
The encrypted messaging service doesn't "contain" this information -- rather the metadata might express it, right? Suggest:

```
For example, the metadata associated with an encrypted messaging service most likely expresses how to initiate the encrypted link before messaging begins.
```

Reply:
Fixed.